app/pages/exportImport/Import.tsx

```diff
@@ -57,9 +59,16 @@ export default function Import() {
             fail('Unable to parse decrypted data!');
             return;
         }
+        if (data.type === mobileWalletExportType) {
+            fail(
+                `Exports from the Concordium Mobile Wallet are not compatible with the Concordium Desktop Wallet, and vice versa.`
```

Comment:
This message states something generic, and is not very direct about what just happened. I would prefix it with:

> The selected file was an export from a Concordium Mobile Wallet. Exports from a Concordium Mobile Wallet [...]